src/BuzzHttpAdapter.php

```diff
@@ -47,6 +47,8 @@ public function __construct(Browser $browser = null, ConfigurationInterface $con
             );
         }
 
+        $browser->getClient()->setVerifyPeer($configuration->getSSLVerifyPeer());
```

Comment:
Why do you prefer that? As I see it, it's a client setting, not a request setting. When it is in `doSendInternalRequest`, it is applied every time, although in fact it is only needed to be set once, or am I wrong?